### ISSUE TITLE
Windows build

### ISSUE THREAD

**Issue by mitvitaminen** [projectM-visualizer/frontend-sdl2#51]
Hi I really made some ppl jealous of projectM and the projectM in Windows Store really does suck is there a build or can you please post a Windows Build? Or can tell me detailed Instructions on how to build one I am affin with IT and Linux but kinda a nap there. 
Because I want to make some people happy and they kinda just stuck in the vicious circle of windows;D and I want them to be able to dream away as I do when watching milkdrop
greets Chris

**Comment by kblaschke** [comment]
Not bleeding-edge build, but this development build here should work fine:

https://github.com/kblaschke/frontend-sdl2/releases/tag/2.0-windows-pre1

Or, use [the Steam app](https://store.steampowered.com/app/1358800/projectM_Music_Visualizer/), it's basically the same as the above build and will be updated when new releases are ready.

Have fun!

**Comment by mitvitaminen** [comment]
Hey Kai first of all thanks for your work really appreciated I just started
to realize how much a nap i am in the windows world nowadays. But anyhow I
am facing following issue with your build: code exec can not proceed
because VCOMP140.DLL was not found. This happens when I execute the App in
the zip folder do I need to install some library first? or is this a bug?
Kind regards Chris

Am Fr., 13. Jan. 2023 um 17:30 Uhr schrieb Kai Blaschke <
***@***.***>:

**Comment by kblaschke** [comment]
Hi Chris, 

it may require the Microsoft Visual C runtime libraries, which aren't always installed.

You can get them directly from Microsoft here:
https://aka.ms/vs/17/release/vc_redist.x64.exe

**Comment by mitvitaminen** [comment]
Moin Kai,
Well thx for the build i could not get it to run yet since the PC is not
available, anyhow I just realized you suggested projectM in steam, which is
free, maybe we should link this as current stablest windows build, because
it is a hustle free steam install? I did not know projectM was available
there so thank you very much for this info. Now I can make a friend's
computer a show off:) I would like them to rather get to know Linux and its
Community, since win11 is quite a change config wise and instead of
learning that one should consider an arch install in my humble opinion.
Anyways, I just wanted to bump you up in this manner. kind regards and thx
again, Chris
Chris

Am So., 22. Jan. 2023 um 20:03 Uhr schrieb Kai Blaschke <
***@***.***>:
